app/models/post.rb

```diff
@@ -2,4 +2,13 @@ class Post < ApplicationRecord
   has_many :comments
   belongs_to :user
```

Comment:
- [ ] The post should belong to an `:author`, not `:user`.

  ```
  belongs_to :author, class_name: 'User'
  ```

- [ ] Just like the comments the post should also have many `:likes`. Kindly add that as well.